server/src/main/java/org/apache/iotdb/openapi/gen/handler/impl/V1ApiServiceImpl.java

```diff
@@ -0,0 +1,1025 @@
+import org.apache.iotdb.db.auth.AuthorityChecker;
+import org.apache.iotdb.db.conf.IoTDBDescriptor;
+import org.apache.iotdb.db.exception.StorageEngineException;
+import org.apache.iotdb.db.exception.metadata.IllegalPathException;
+import org.apache.iotdb.db.exception.metadata.MetadataException;
+import org.apache.iotdb.db.exception.metadata.StorageGroupNotSetException;
+import org.apache.iotdb.db.exception.query.QueryProcessException;
+import org.apache.iotdb.db.metadata.PartialPath;
+import org.apache.iotdb.db.qp.Planner;
+import org.apache.iotdb.db.qp.executor.PlanExecutor;
+import org.apache.iotdb.db.qp.physical.PhysicalPlan;
+import org.apache.iotdb.db.qp.physical.crud.GroupByTimePlan;
+import org.apache.iotdb.db.qp.physical.crud.QueryPlan;
+import org.apache.iotdb.db.qp.physical.sys.ShowChildPathsPlan;
+import org.apache.iotdb.db.qp.physical.sys.ShowPlan.ShowContentType;
+import org.apache.iotdb.db.query.context.QueryContext;
+import org.apache.iotdb.db.query.control.QueryResourceManager;
+import org.apache.iotdb.db.query.executor.QueryRouter;
+import org.apache.iotdb.openapi.gen.handler.NotFoundException;
+import org.apache.iotdb.openapi.gen.handler.V1ApiService;
+import org.apache.iotdb.openapi.gen.model.GroupByFillPlan;
+import org.apache.iotdb.openapi.gen.model.ReadData;
+import org.apache.iotdb.openapi.gen.model.WriteData;
+import org.apache.iotdb.tsfile.exception.filter.QueryFilterOptimizationException;
+import org.apache.iotdb.tsfile.read.common.Field;
+import org.apache.iotdb.tsfile.read.common.Path;
+import org.apache.iotdb.tsfile.read.common.RowRecord;
+import org.apache.iotdb.tsfile.read.query.dataset.QueryDataSet;
+
+import com.google.common.base.Joiner;
+import com.google.gson.Gson;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.ProtocolStringList;
+import org.apache.commons.lang3.StringUtils;
+import org.xerial.snappy.Snappy;
+import prometheus.Remote;
+import prometheus.Remote.Query;
+import prometheus.Remote.ReadRequest;
+import prometheus.Remote.ReadResponse;
+import prometheus.Types;
+import prometheus.Types.Label;
+import prometheus.Types.LabelMatcher;
+import prometheus.Types.Sample;
+import prometheus.Types.TimeSeries;
+import prometheus.Types.TimeSeries.Builder;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@javax.annotation.Generated(
+    value = "org.openapitools.codegen.languages.JavaJerseyServerCodegen",
+    date = "2021-01-13T21:45:03.765+08:00[Asia/Shanghai]")
+public class V1ApiServiceImpl extends V1ApiService {
```

Comment:
why  V1?

Reply:
The method name is the code automatically generated from the configuration file  iotdb-rest.yaml